### PR TITLE
SDK-275 : Add age verification helper

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

File: examples/profile.php [hunk 1]
```diff
@@ -17,7 +17,9 @@
     $base64Selfie = ActivityDetailsHelper::getBase64Selfie($profile);
     // This key uses the  format: age_[over|under]:[1-999] and is dynamically
```
Comment (vassyz) [inline examples/profile.php]: Remove this comment

File: examples/profile.php [hunk 2]
```diff
@@ -17,7 +17,9 @@
...
     // generated based on the dashboard attribute Age / Verify Condition
-    $ageVerified = $profile->getProfileAttribute('age_over:18');
+    $ageVerified = $profile->isAgeVerified() ? 'true' : '';
+    $verifiedAge = $profile->getVerifiedAge();
```
Comment (vassyz) [inline examples/profile.php]: Maybe add a comment of what this does?

File: examples/profile.php [hunk 3]
```diff
@@ -17,7 +17,9 @@
     $base64Selfie = ActivityDetailsHelper::getBase64Selfie($profile);
     // This key uses the  format: age_[over|under]:[1-999] and is dynamically
     // generated based on the dashboard attribute Age / Verify Condition
-    $ageVerified = $profile->getProfileAttribute('age_over:18');
+    $ageVerified = $profile->isAgeVerified() ? 'true' : '';
```
Comment (vassyz) [inline examples/profile.php]: What does this do? Are you converting it to a string or not wanting to display `false`?

File: tests/Util/Age/AgeUnderOverProcessorTest.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    public $processor;
+    public $dummyProfile;
+
+    public function setUp()
+    {
+        $this->dummyProfile = [
+            'family_name' => 'TestFamilyName',
+            'given_names' => 'TestGivenName',
+            'full_name'   => 'TestGivenName TestFamilyName',
+            'date_of_birth' => '11-07-2017',
```
Comment (vassyz) [inline tests/Util/Age/AgeUnderOverProcessorTest.php]: I know it doesn't matter, but can you add a date so the fake profile is over 18?

File: tests/Util/Age/ProcessorTest.php [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+     */
+    public $processor;
+
+    public function setUp()
+    {
+        $dummyProfile = [
+            'family_name' => 'TestFamilyName',
+            'given_names' => 'TestGivenName',
+            'full_name'   => 'TestGivenName TestFamilyName',
+            'date_of_birth' => '11-07-2017',
```
Comment (vassyz) [inline tests/Util/Age/ProcessorTest.php]: Same here, age > 18

File: examples/profile.php [hunk 6]
```diff
@@ -17,7 +17,14 @@
     $base64Selfie = ActivityDetailsHelper::getBase64Selfie($profile);
     // This key uses the  format: age_[over|under]:[1-999] and is dynamically
     // generated based on the dashboard attribute Age / Verify Condition
-    $ageVerified = $profile->getProfileAttribute('age_over:18');
+    $ageVerified = $profile->isAgeVerified();
+    if(is_bool($ageVerified)) {
```
Comment (vassyz) [inline examples/profile.php]: This conditional doesn't look alright to me. Let's look into it a bit more.

Comment (sidibos) [reply examples/profile.php]: This function will return two types a boolean `TRUE or FALSE` if the attribute is set or a string `NULL` if the attribute is not set.

**Pull request merged**